docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
 ---
-title: "PingSource"
+title: "PingSource example"
 linkTitle: "PingSource"
-weight: 31
+weight: 10
 type: "docs"
+aliases:
+  - ../cronjob-source
```

Comment:
I think this needs to be an absolute link, e.g. /docs/whatever/path

Reply:
Given no alias was present before (I think it was added through some merging accident) I'll simply delete it.